src/main/java/sirius/db/mixing/BaseEntity.java

```diff
@@ -66,12 +66,43 @@
     /**
      * Returns the descriptor which maps the entity to the database table.
      *
-     * @return the ddescriptor which is in charge of checking and mapping the entity to the database
+     * @return the descriptor which is in charge of checking and mapping the entity to the database
      */
     public EntityDescriptor getDescriptor() {
         return mixing.getDescriptor(getClass());
     }
 
+    /**
+     * Builds a standard message that the given entity cannot be saved.
+     * <p>
+     * This message is mostly used as a consequence describing why an error happened.
+     * For instance, when an entity's required field is not populated, a standard message like this will be logged:
+     * {@code Field 'MyField' is required.}
+     * As to would be rather nice to tell the end user that an entity couldn't be saved due to the error.
+     * {@code MyEntity cannot be saved. Field 'MyField' is required.}
+     *
+     * @param clazz the entity class to extract its label
+     * @return the formatted message
+     */
+    public static String createCannotSaveMessage(@Nonnull Class<?> clazz) {
```

Comment:
Do we really need the static implementation?

Reply:
We do, because we must build this message for logging even before an entity even comes to exist. For example in context of ImportHandlers.

Reply:
move to descriptor?